gpkit/constraints/array.py

```diff
@@ -0,0 +1,12 @@
+"Implements ArrayConstraint"
+from .set import ConstraintSet
+from .single_equation import SingleEquationConstraint
+
+
+class ArrayConstraint(ConstraintSet, SingleEquationConstraint):
+    "A ConstraintSet for prettier array-constraint printing"
+    def __init__(self, constraints, left, oper, right):
```

Comment:
more docstring would help here -- reading it for the first time, it's not obvious what types should be passed for `constraints`, `left`, and `right`.